controllers/openshift_controller.go

```diff
@@ -218,6 +239,37 @@ func (r *KataConfigOpenShiftReconciler) processDaemonsetForMonitor() *appsv1.Dae
 	}
 }
 
+func (r *KataConfigOpenShiftReconciler) processDashboardConfigMap() *corev1.ConfigMap {
+
+	r.Log.Info("Creating sandboxed containers dashboard in the OpenShift console")
+	cmName := "grafana-dashboard-sandboxed-containers"
+	cmLabels := map[string]string{
+		"console.openshift.io/dashboard": "true",
+	}
+	cmNamespace := "openshift-config-managed"
+
+	// retrieve content of the dashboard from our own namespace
+	foundCm := &corev1.ConfigMap{}
+	err := r.Client.Get(context.TODO(), types.NamespacedName{Name: cmName, Namespace: "openshift-sandboxed-containers-operator"}, foundCm)
+	if err != nil {
+		r.Log.Error(err, "could not get dashboard data")
+		//return nil
```

Comment:
This line must not be commented. I did it for troubleshooting, and did not remove it.
If we leave it, having an error here would result in creating an empty ConfigMap, which actually breaks the "Observe / Dashboard" panel in the OCP console.